### PR TITLE
Bisect voc

### PR DESCRIPTION
**Pull request by pranitbauva1997**
I have converted the function bisect_voc().

I am currently working on bisect_clean_state(). Will finish this soon.

### CODE REVIEW

File: builtin/bisect--helper.c [hunk 1]
```diff
@@ -79,18 +79,31 @@ int write_terms(const char *bad, const char *good)
 	strbuf_release(&content);
 	return (res < 0) ? -1 : 0;
 }
+
+int bisect_voc(const char *term){
+	if (!strcmp(term, "bad"))
+		printf("bad|new");
```
Comment (chriscool) [inline builtin/bisect--helper.c]: Please also print a new line character, as echo does.

Comment (pranitbauva1997) [inline builtin/bisect--helper.c]: Sure! Will update!

File: builtin/bisect--helper.c [hunk 2]
```diff
@@ -79,18 +79,31 @@ int write_terms(const char *bad, const char *good)
 	strbuf_release(&content);
 	return (res < 0) ? -1 : 0;
 }
+
+int bisect_voc(const char *term){
+	if (!strcmp(term, "bad"))
+		printf("bad|new\n");
+	else if(!strcmp(term, "good"))
+		printf("good|old\n");
+
```
Comment (chriscool) [inline builtin/bisect--helper.c]: You could remove the blank line here.

Comment (pranitbauva1997) [inline builtin/bisect--helper.c]: Sure! Actually I forgot to insert in write-terms branch and sent the patch. So instead did it here.

**Pull request closed without merging**